README.md

```diff
@@ -266,6 +266,15 @@ If you see SQL errors after upgrading please remember to check for this specific
 
 UPS shut down their old CGI APIs so we removed the support for it from the Mage_Usa module.
 
+### Between OpenMage 20.x and 21.x (unreleased, available on branch `next`)
+
+- PHP 8.1 is now the minimum version
+- Removed scriptaculous/dragdrop.js (#3215)
+- RWD theme: updated jQuery to 3.7.0 (#3204)
+- Unified CSRF configuration (#3147) and added form key validation to Contacts form (#3146)
+- Removed double span element from HTML buttons (#3123)
+- Removed all deprecated Mysql4_ classes (#2730), if you have old modules that still use those classes, please use the 'shell/rename-mysql4-class-to-resource.php` to convert them
```

Comment:
Removed all deprecated Mysql4_ classes (#2730)**. If you have installed old modules/extensions that use such classes then you must run in CLI /{OpenMage_ROOT}/shell/rename-mysql4-class-to-resource.php to convert them. Please backup all files before running the script**